packages/core/src/components/pagination/pagination.tsx

```diff
@@ -40,6 +55,19 @@ export class Pagination implements ComponentInterface {
     });
   };
 
+  componentDidLoad(): void {
+    const responsiveElement = this.host.shadowRoot?.querySelector('dso-responsive-element');
+    responsiveElement?.getSize().then((size: string) => this.elementPositions = this.sizePositionsMap[size]);
+
+    this.resizeObserver = new ResizeObserver(() => responsiveElement?.getSize().then((size: string) => this.elementPositions = this.sizePositionsMap[size]));
```

Comment:
Dit gedrag is passender om bij `<dso-responsive-element>` onder te brengen. Daar bestaat al een resizeObserver.

Zie ook de description van het issue:
> `<dso-responsive-element>` breiden we uit met een @Event() size event emitter. Bij het wijzigen van de size, emit hier de sizeAlias.

In componentDidLoad hang je dan aan dat event, en bij disconnected unsubscribe je de listener.

edit: Ik zie nu ook dat je wel netjes het `@Event` hebt gemaakt, je gebruikt 'm alleen nog niet :(

Reply:
Ik denk dat je ook met Refs kunt werken ipv `this.host.querySelector`

Reply:
`@Listen` op `dsoSizeChange` toegevoegd en ref gebruikt.

Reply:
Vet 👍